pandas/core/groupby/generic.py

```diff
@@ -2303,7 +2303,7 @@ def value_counts(
         normalize : bool, default False
             Return proportions rather than frequencies.
         sort : bool, default True
-            Sort by frequencies.
+            Sort by frequencies when True. Sort by DataFrame column values when False.
```

Comment:
I think we also want to modify the docstring on `SeriesGroupBy.value_counts`. Can you make the same adjustment there.

Reply:
Sure thing